TMP2022/technical-writing-track/precious-tw-funke/README.md

```diff
@@ -0,0 +1,239 @@
+# Kuda Bank User Guide For Android Users
+
+Nigeria's first mobile-only bank, [Kuda Bank](https://kuda.com/), was granted a license by the Central Bank of Nigeria (CBN) in 2019. Consequently, Kuda is only available online and has no physical locations. Babs Ogundeyi and Musty Mustapha formed Kuda Bank. This guide illustrates getting started with the Kuda bank app.
+
+We'll go over creating a Kuda account and using the app's fundamental features. Download your app on the [Google Playstore](https://play.google.com/store/apps/details?id=com.kudabank.app&hl=en&gl=US) store for android devices and Let's get started.
+
+# CREATE ACCOUNT
+
+You have control over your finances, spending, and savings when you have a Kuda account. You may accomplish all of this with the assistance of Kuda, commonly known as the bank of the free, with almost no charges. Let's look at creating a Kuda account.
+
+## Open An Account.
```

Comment:
Break these down into steps and add images for each step

Reply:
Did an update with this in mind.